specs/rollup-node.md

```diff
@@ -268,6 +270,25 @@ previous section to each successive L1 block until we have caught up with the L1
 
 > **TODO** specify genesis block
 
+## L2 Output RPC method
+
+The Rollup node has its own RPC method, `optimism_output` which returns the
+a 32 byte hash corresponding to the [SSZ] encoded [L2Output](./proposals.md#l2-output-commitment-construction).
+
+[SSZ]: https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md
+
+### Output Method API
```

Comment:
This is pretty short, really worth splitting into a sub-section?

Reply:
I'm fine with it.